Improve Chainguard Libraries for Python docs (#2278)
chainguard-dev/edu: content/chainguard/libraries/python/overview.md

```
https://libraries.cgr.dev/python/
```

Use the URL with your [username and password retrieved with
chainctl](/chainguard/libraries/access/) to access the Chainguard Libraries for
Python repository manually with a browser.

After successful login, you see a message `Forbidden`. Following PyPI URL
conventions, a browsable directory structure can be viewed at the `simple`
sub-context at `https://libraries.cgr.dev/python/simple/`. The top level
contains a list of packages:

```
2captcha-python
3d-converter
absql
ahrs
amqpstorm
annogesic
apiflask
apscheduler
...
```

A list of all wheels and tarballs for the versions of a specific package is
available in the context of the package. For example, the `apiflask` context at
`https://libraries.cgr.dev/python/simple/apiflask/` shows the following list:

```
Links for apiflask
apiflask-0.1.0-py3-none-any.whl
apiflask-0.1.0.tar.gz
apiflask-0.10.0-py3-none-any.whl
apiflask-0.10.0.tar.gz
apiflask-0.10.1-py3-none-any.whl
apiflask-0.10.1.tar.gz
apiflask-0.11.0-py3-none-any.whl
apiflask-0.11.0.tar.gz
apiflask-0.12.0-py3-none-any.whl
apiflask-0.12.0.tar.gz
...
```

Each package name is a link with to the specific binary. The link includes long

> unique identifiers and can not be determined without browsing. The list uses

Reviewer: ```suggestion
unique identifiers and cannot be determined without browsing. The list uses
```
